howso/client/cache.py

```diff
@@ -11,7 +13,13 @@ class TraineeCacheItem(t.TypedDict):
     """Type definition for trainee cache items."""
 
     trainee: Trainee
+    """Trainee object."""
```

Comment:
Curious. Why not just a comment? This is executed at runtime and declares a string.

Reply:
This is a docstring comment?